packages/legend-query-builder/src/stores/explorer/QueryBuilderPropertySearchState.ts

```diff
@@ -317,17 +505,27 @@ export class QueryBuilderPropertySearchState {
     this.isSearchPanelHidden = val;
   }
 
+  setIsSearchPanelTipsOpen(val: boolean): void {
+    this.isSearchPanelTipsOpen = val;
+  }
+
+  setIsOverSearchLimit(val: boolean): void {
+    this.isOverSearchLimit = val;
+  }
+
   setSearchText(val: string): void {
     this.searchText = val;
   }
 
   setSearchedMappedPropertyNodes(
     val: QueryBuilderExplorerTreeNodeData[],
   ): void {
+    this.searchState.inProgress();
```

Comment:
I'm not sure but this looks misused, `action state` is meant for long-running processes. What are you trying to do here?

Reply:
yes, wonder what I was trying to do here...removed 🥲